pkg/api/graph/graph.go

```diff
@@ -41,8 +41,12 @@ func (n UniqueName) UniqueName() string {
 	return string(n)
 }
 
+func (n UniqueName) String() string {
+	return string(n)
+}
+
 type uniqueNamer interface {
-	UniqueName() string
+	UniqueName() UniqueName
```

Comment:
This change looks good.  I suspect this was the intent and it broke at some point.